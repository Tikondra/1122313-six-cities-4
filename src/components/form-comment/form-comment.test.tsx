import * as React from "react";
import * as renderer from "react-test-renderer";
import FormComment from "./form-comment";

it(`should render FormComment`, function () {
  const tree = renderer
    .create(
        <FormComment comment={``} isBlocked={false} onChangeComment={jest.fn}/>,
        {createNodeMock: () => {
          return document.createElement(`div`);
        }}
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
