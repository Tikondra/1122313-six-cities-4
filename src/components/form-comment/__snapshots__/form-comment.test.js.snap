// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FormComment 1`] = `
<textarea
  className="reviews__textarea form__textarea"
  disabled={false}
  id="review"
  maxLength={300}
  minLength={50}
  name="review"
  onChange={[Function]}
  placeholder="Tell how was your stay, what you like and what can be improved"
  value=""
/>
`;
